packages/client/components/TeamlineEventTypeMenu.tsx

```diff
@@ -0,0 +1,70 @@
+import React, {useMemo} from 'react'
+import useAtmosphere from '~/hooks/useAtmosphere'
+import useRouter from '~/hooks/useRouter'
+import constructFilterQueryParamURL from '~/utils/constructFilterQueryParamURL'
+import {useQueryParameterParser} from '~/utils/useQueryParameterParser'
+import {MenuProps} from '../hooks/useMenu'
+import DropdownMenuLabel from './DropdownMenuLabel'
+import Menu from './Menu'
+import MenuItem from './MenuItem'
+import TimelineEventTypeEnum from '../../server/graphql/types/TimelineEventTypeEnum'
+import EventTypeFilterMenuItemLabel from './EventTypeFilterMenuItemLabel'
+
+interface Props {
+  menuProps: MenuProps
+}
+
+const TeamlineEventTypeMenu = (props: Props) => {
+  const {history} = useRouter()
+  const {menuProps} = props
+  const atmosphere = useAtmosphere()
+  const eventTypes = TimelineEventTypeEnum.getValues().map(({value}) => value)
+  const {
+    eventTypes: eventTypeFilters,
+    teamIds,
+    userIds
+  } = useQueryParameterParser(atmosphere.viewerId)
+
+  const showAllEvents = !!eventTypeFilters
+  const {eventTypeValues, defaultActiveIdx} = useMemo(() => {
+    const eventTypeValues = eventTypes
+    return {
+      eventTypeValues,
+      defaultActiveIdx:
+        eventTypeValues.findIndex((eventType) => eventTypeFilters?.includes(eventType)) +
+        (showAllEvents ? 2 : 1)
+    }
+  }, [eventTypeFilters])
+
+  return (
+    <Menu
```

Comment:
-1 two small UI things:

1. A vertical scroll bar

![vertical-scroll-bar](https://github.com/ParabolInc/parabol/assets/39854876/e69f6b51-4ab0-4844-ad54-20b7615d282e)


2. There's not quite enough space for the final menu item. I think it need some extra padding to match to first menu item 
![Screenshot 2023-07-03 at 16 35 58](https://github.com/ParabolInc/parabol/assets/39854876/2197f671-753e-4ff9-a8cc-fe02551ee156)

Reply:
These are existing issues: if you have 5 teams and open the team filter in team dash or user dash, you will see the vertical scrolling bar & no padding for the last menu item. I think this has something to do with the `Menu` or `MenuItem` components that are reused across the app. I feel like fixing it is out of the scope of this PR. WDYT?

Reply:
Yeah, I agree - it looks like an existing issue, so I think it's out of scope for this PR. A GitHub issue would be helpful though

Reply:
Created an issue [here](https://github.com/ParabolInc/parabol/issues/8500)